api/src/pcapi/core/users/commands.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from pcapi import settings
+import pcapi.core.fraud.api as fraud_api
+from pcapi.core.users import api
+import pcapi.core.users.constants as users_constants
+import pcapi.scheduled_tasks.decorators as cron_decorators
+from pcapi.utils.blueprint import Blueprint
+
+
+blueprint = Blueprint(__name__, __name__)
+logger = logging.getLogger(__name__)
+
+
+@blueprint.cli.command("notify_users_before_deletion_of_suspended_account")
+@cron_decorators.log_cron_with_transaction
+def notify_users_before_deletion_of_suspended_account() -> None:
+    api.notify_users_before_deletion_of_suspended_account()
+
+
+@blueprint.cli.command("delete_suspended_accounts_after_withdrawal_period")
```

Comment:
cette fonction a été déplacé ici depuis api/src/pcapi/scheduled_tasks/commands.py

Reply:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏cool de déplacer ça
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Au lieu d'un commentaire dans la PR, fais un commit séparé pour ce genre de modifs